toolchest_client/files/unpack.py

```diff
@@ -0,0 +1,77 @@
+from distutils.dir_util import copy_tree
+from enum import Enum
+import os
+from pathlib import Path
+import shutil
+
+
+class OutputType(Enum):
+    GZ_TAR = ".tar.gz"
+    FLAT_TEXT = ".txt"
```

Comment:
If we ever want to make the returned filetype a customizable parameter, we should make it easy to use (e.g., infer the output type from the output path). This is just something to keep in mind for the future, though.